### PR TITLE
Added man pages for tools

### PR DESCRIPTION
**Pull request by Mailaender**
from the Debian project.

### CODE REVIEW

**Comment by Mailaender** [comment]
Test installation at https://build.opensuse.org/request/show/349740

File: s2tc_from_s3tc.1 [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.TH S2TC_FROM_S3TC 1 "April 7, 2012"
+.\" Please adjust this date whenever revising the manpage.
+.SH NAME
+s2tc_from_s3tc \- Convert a S3TC format texture into S2TC format one.
+.SH SYNOPSIS
+\fBs2tc_from_s3tc\fP 
+.SH DESCRIPTION
+s2tc_from_s3tc is part of the S2TC toolset. It allows conversion of 
+S3TC format textures into S2TC format ones using the lossy algorithm
+implemented in the algorithm.
```
Comment (divVerent) [inline s2tc_from_s3tc.1]: ... Implemented in S2TC (algorithm in the algorithm is weird)

**Comment by divVerent** [comment]
Apart from the small wording issue, well done - a man page sounds useful here.

Will merge once the double algorithm in the wording is fixed.

**Comment by divVerent** [comment]
Actually - can you also please warn that s2tc_from_s3tc is faster but yields lower quality than decompressing using a s3tc tool then compressing using s2tc - is is even worse than decompressing using s2tc actually?

This tool is really a very simple conversion without any kind of dithering and mainly exists to get a quick upper bound estimate of the quality loss (IIRC I wrote it before he actual S2TC algorithm to get a quick overview of how bad it will be).

**Comment by Mailaender** [comment]
Rebased and updated. Also removed some trailing spaces and redundant wordings.

**Pull request merged**